docs/api-reference/static-analysis.md

```diff
@@ -83,66 +84,134 @@ Finally, to make `eslint` easy to run, it is suggested to place commands in `pac
 * `dbosRecommendedConfig` - This extension to the base configuration is the recommended configuration, which includes best practices for new TypeScript code.
 * `dbosExtendedConfig` - This configuration extenends the recommended configuration, and may include opinions on coding style.
 
-### Suggested `eslint` Rules
+---
+
+### Base rules/plugins
```

Comment:
Get rid of this section, we don't need to document ESLint, just list our entire recommended config and link each rule to its documentation

Reply:
Resolved